### PR TITLE
[CUBRIDQA-72] refine the format and  the content of readme

### PR DESCRIPTION
**Pull request by supler**

### CODE REVIEW

**Comment by mandy2016** [comment]
CUBRID and CUBRID_DATABASES environment variables should be configured before executing testing, please refer to http://www.cubrid.org/ for configurations
-->  for the configurations 

article(a, an, the) is always needed before a noun except a very few specific conditions.

**Comment by mandy2016** [comment]
For Medium test, data_file must be configured into the [sql] section of conf file with the path of the initial data file
-->
For Medium test, data_file must be configured **in** the [sql] section of **the** conf file with the path of the initial data file

**Comment by mandy2016** [comment]
For SQL_By_CCI test, sql_interface_type=cci must be configured into the [sql] section of conf file
-->
For SQL_By_CCI test, sql_interface_type=cci must be configured **in** the [sql] section of **the** conf file

**Comment by mandy2016** [comment]
Regarding more explanation for parameters setting, please refer to CTP/conf/sql.conf for SQL, CTP/conf/medium.conf for Medium
-->
**For** more **explanations** **of** parameters setting, please refer to CTP/conf/sql.conf for SQL, CTP/conf/medium.conf for Medium

**Comment by mandy2016** [comment]
When it is completed, CTP will print the summary result message
-->(it is not clear here)
When **the test** is completed, CTP will print the summary result message

**Comment by jiyanping** [comment]
if you are sure the port will not conflict, just ignore.
--> I think it will be better to use the below description
If you're sure there would be no conflict, please ignore it.

**Comment by mandy2016** [comment]
You will have an output as follows:

$ bin/ctp.sh webconsole start
Config: /home/user/CTP/conf/webconsole.conf
Web Root: /home/user/CTP/sql/webconsole
Begin to start ...

Done
URL:  http://127.0.0.1:8888    
-->(the output is not exactly the same, so 'like' is better)
You will have an output like this:

File: CTP/README.md [hunk 1]
```diff
@@ -271,42 +278,44 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
...
-
-      ```
-	  # Define the URL of test build which will be used to test
-	  # If this parameter is not set or commented out, CTP will execute testing without build installation.
-	  cubrid_download_url=http://127.0.0.1/download/CUBRID-10.1.0.6929-b049ba5-Linux.x86_64.sh 
-	  ```
-
-	 Regarding more explanation for parameters setting, please refer to [CTP/conf/ha_repl.conf](conf/ha_repl.conf)	
-	  
+	* Prepare at least two accounts for the multiple test instances (e.g., one account named as ``controller``, another account named as ``ha_repl_instance1``)
```
Comment (jinmeihua) [inline CTP/README.md]: Actually, HA replication test needs at least three accounts: one controller , one master and one slave.

**Comment by mandy2016** [comment]
On shell_instance1 account
-->
On instance1 node

**Comment by fanzaiqiang** [comment]
@supler 

Please go ahead to commit. Thanks a lot.

**Pull request merged**